pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md

```diff
@@ -0,0 +1,182 @@
+---
+$category@: media
+formats:
+  - websites
+  - stories
+  - ads
+teaser:
+  text: 替代 HTML5 audio 标记。
+toc: true
+$title: amp-audio
+version: '0.1'
+versions:
+  - '0.1'
+latest_version: '0.1'
+is_current: true
+$path: /documentation/components/amp-audio.html
+$localization:
+  path: '/{locale}/documentation/components/amp-audio.html'
+layouts:
+  - fixed
+  - fixed-height
+  - nodisplay
+scripts:
+  - >-
+    <script async custom-element="amp-audio"
+    src="https://cdn.ampproject.org/v0/amp-audio-0.1.js"></script>
```

Comment:
Please remove all keys that are not relevant for translation. I.e. only keep `teaser`, all other keys will then cascade to the original document. 🙂

Reply:
done